test_runner/regress/data/test_remote_extensions/test_extension/test_extension.control

```diff
@@ -0,0 +1 @@
+comment = 'Test extension'
```

Comment:
Ah, that's nice! My only concern is that we don't do tests with any extension with preload libs. I wonder, can we do some trick with one of contrib extensions? Like compile it, but not install, then declare as a remote extension. Could be a separate PR, though, not sure we need to block on it

Reply:
Yeah, you bring up a good point. Let me create an issue and we can get to it when the team has time for "busy" work :laughing:

Reply:
https://github.com/neondatabase/neon/issues/11146